developer/src/common/web/utils/src/types/kps/kps-file.ts

```diff
@@ -91,22 +91,20 @@ export interface KpsFileLanguages {
 }
 
 export interface KpsFileLanguage {
-  _: string;
-  $: { ID: string }
+  '#text': string;
+  ID: string;
 }
 
 export interface KpsFileRelatedPackages {
   RelatedPackage: KpsFileRelatedPackage | KpsFileRelatedPackage[];
 }
 
 export interface KpsFileRelatedPackage {
-  $: {
-    ID: string;
-    /**
-     * relationship between this package and the related package, "related" is default if not specified
-     */
-    Relationship?: "deprecates";
-  }
+  ID: string;
+  /**
+   * relationship between this package and the related package, "related" is default if not specified
+   */
+  Relationship?: "deprecates";
```

Comment:
Shouldn't this be

```suggestion
  Relationship?: "deprecates"|"related";
```

if the default is "related"? (I didn't check if there are other possible values)

Reply:
Currently only `"deprecates"` is allowed:

https://github.com/keymanapp/keyman/blob/400444933cb51e972d7c9b95fbaa06b520d8feb6/common/schemas/kps/kps.xsd#L108-L109